docs/algorithms.rst

```diff
@@ -20,7 +25,8 @@ YANK provides the somewhat unique ability to perform alchemical free energy calc
 Solvent model
 ^^^^^^^^^^^^^
 
-For the ``yank prepare amber`` command that imports AMBER ``prmtop`` files, any of the AMBER implicit solvent models available in OpenMM are available for use via the :code:`--gbsa <model>`` argument:
+For the ``yank prepare amber`` command that imports AMBER ``prmtop`` files, any of the AMBER implicit solvent models
```

Comment:
I think we want to ditch the `yank prepare amber` stuff totally. It will just cause confusion.